sdk/communication/communication-call-automation/test/callMediaClient.spec.ts

```diff
@@ -46,7 +74,7 @@ describe("CallMedia Unit Tests", () => {
 
   it("PlayToAll", async () => {
     const playSource: FileSource = {
-      uri: "https://example.com/audio/test.wav",
+      uri: "https://acstestapp1.azurewebsites.net/audio/bot-hold-music-1.wav",
```

Comment:
Can't we use a dummy url here like before the change? These sensitive info should not be on the repo though.